src/components/AssistanceForm/AssistanceForm.tsx

```diff
@@ -31,18 +29,25 @@ export const AssistanceForm = () => {
   async function getAssistanceJWT(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) {
     e.preventDefault();
     try {
-      const { data: zendeskAssistance } = await utils.apiClient.token.getZendeskAssistanceToken({
-        userEmail: email
-      });
+      await utils.apiClient.token
+        .getZendeskAssistanceToken({
+          userEmail: email
+        })
+        .then((data) => {
+          const { data: zendeskAssistance } = data;
+          const resultData = zendeskAssistanceTokenResponseSchema.safeParse(zendeskAssistance);
+          if (!resultData.success) throw resultData.error;
+          if (zendeskAssistance.assistanceToken == '' || zendeskAssistance.returnTo == '')
```

Comment:
Should we check for whitespaces?
```javascript
if (!zendeskAssistance.assistanceToken.trim() || !zendeskAssistance.returnTo.trim())
```

Reply:
No more. Now the BE has fix a old problem with white spaces :-)